src/MessagePack/Formatters/PrimitiveObjectFormatter.cs

```diff
@@ -67,6 +67,36 @@ public static bool IsSupportedType(Type type, TypeInfo typeInfo, object value)
             return false;
         }
 
+        public static bool IsSupportedType(Type type, object value)
```

Comment:
Rather than merely duplicate this method for the `Type` overload, how about revising the original method to forward its call into this new overload so there's only one implementation? `TypeInfo` derives from `Type` so it should be directly callable.